tests/lib/bin/download-file

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env any-python
+
+from argparse import ArgumentParser
+import requests
+import os
+try:
+    from urlparse import urlsplit
+except ImportError:
+    from urllib.parse import urlsplit
+
+
+MAX_ATTEMPTS = 5
+CONNECTION_TIMEOUT = 10
+READ_TIMEOUT = 5
```

Comment:
it would be nice if there was an easy way to override these parameters from the command line, especially for a developer trying to debug if a particular endpoint is just really slow. Unfortunately I have seem some internet connections with really low latency that while slow would eventually pass if I could have configured a timeout like this was just a little bit higher and be a little bit more tolerant *cough* *cough* hotel WiFi